addons/l10n_ec/data/account_tax_report_data.xml

```diff
@@ -87,6 +87,42 @@
                             </record>
                         </field>
                     </record>
+                    <record id="tax_report_line_parent_line_report_7" model="account.report.line">
+                        <field name="name">Ventas locales (excluye activos fijos) gravadas tarifa 5%</field>
```

Comment:
please note that starting from the next version in the forward port (saas-16.3) these report lines should be in English in the XML file and then we should add translations for these lines